networkit/cpp/io/NetworkitBinaryReader.cpp

```diff
@@ -148,24 +157,81 @@ Graph NetworkitBinaryReader::read(const std::string& path) {
 			//Read adjacency lists.
 			for (uint64_t j = 0; j < outNbrs; j++) {
 				uint64_t add;
+				double weight;
 				off += NetworkitBinaryReader::decode(reinterpret_cast<const uint8_t*>(adjIt + off), add);
+				switch(weightFormat) {
+					case nkbg::WEIGHT_FORMAT::unsignedFormat:
+						{
+							uint64_t unsignedWeight;
+							wghtOff += NetworkitBinaryReader::decode(reinterpret_cast<const uint8_t*>(adjWghtIt + wghtOff), unsignedWeight);
+							weight = unsignedWeight;
+						}
+					break;
```

Comment:
The `break` is part of the `case`, indent it accordingly. Furthermore, the double indentation is unnecessary. Indent the `case` as follows:
```c
switch (baz) {
    case foo: {
        bar();
        break;
    }
    [...]
}
```